.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDatePeriodParser.cs

```diff
@@ -705,8 +858,8 @@ private DateTimeResolutionResult MergeTwoTimePoints(string text, DateObject refe
             {
                 pastEnd = futureEnd;
             }
-            
-            ret.Timex = $"({pr1.TimexStr},{pr2.TimexStr},P{(futureEnd - futureBegin).TotalDays}D)";
```

Comment:
Is this causing some side-effect?
Can we add some test cases for periods like 1.5 days and see if things behave as expected?

Reply:
I just checked, it do have some side effects for cases like "I will stay here from today to two and a half days later". I will change this back.

Reply:
Add add one test case to avoid regression in the future.